sdk/cosmosdb/cosmos/src/client/SasToken/DataPlanePermissionScope.ts

```diff
@@ -0,0 +1,90 @@
+// Copyright (c) Microsoft Corporation.
```

Comment:
Same as container permissions scopes. I think this file can be removed entirely https://github.com/Azure/azure-sdk-for-js/commit/ab736c194b2f8b6a2ec890cdbc91c4e55d071785